src/ipfs.io/docs/install/index.html

```diff
@@ -10,118 +10,91 @@
 <div class="col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2" style="font-family: Oxygen; line-height: 1.7">
 {% markdown %}
 
-# Install IPFS
+# Install Go IPFS
 
-You can install the [Go IPFS implementation](https://github.com/ipfs/go-ipfs)
-from source or from prebuilt packages.
+We recommend to install IPFS from a prebuilt package. You can obtain those from [dist.ipfs.io](http://dist.ipfs.io/#go-ipfs).
 
+For convenience here are the latest versions (`0.3.11`) linked directly
+- <i class="fa fa-apple"></i> [Mac OS X]
+- <i class="fa fa-linux"></i> [Linux 32bit]
+- <i class="fa fa-linux"></i> [Linux 64bit]
+- <i class="fa fa-windows"></i> [Windows 32bit]
+- <i class="fa fa-windows"></i> [Windows 64bit]
```

Comment:
Let's keep links like these inlined. Separating information like this (putting the references and the links in separate places of the document) does not play well with VCS and all sorts of other tools to manipulate documents.